src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -311,7 +313,7 @@ private void GroupDescriptionTextBox_OnIsVisibleChanged(object sender, Dependenc
             //Record the value here, this is useful when title is popped from stack during undo
             ViewModel.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(
                 new DynCmd.UpdateModelValueCommand(
-                    Guid.Empty, ViewModel.AnnotationModel.GUID, "GroupNameTextBlockText",
+                    Guid.Empty, ViewModel.AnnotationModel.GUID, "GroupDescriptionTextBlockText",
```

Comment:
Is there a way to get this string from somewhere? If not, please ignore this comment :)